frontend/src/components/StartMenuComponent/StartMenuComponent.test.tsx

```diff
@@ -0,0 +1,24 @@
+import { StartMenuComponent } from "./StartMenuComponent";
+import {
+  render,
+  screen,
+} from "@testing-library/react";
+
+describe("Start Menu Component", () => {
+  it(`should show title "TourDeFoos"`, () => {
+    //Given
+    //When
+    render(<StartMenuComponent  />);
+    //Then
+    expect(screen.getByText("TourDeFoos")).toBeInTheDocument();
+  });
+
+  it(`should show buttons with text: "Turniej", "Zawodnicy"`, () => {
+    //Given
+    //When
+    render(<StartMenuComponent  />);
+    //Then
+    expect(screen.getByText("Turniej")).toBeInTheDocument();
```

Comment:
Hmm... sprwadziłbym jeszcze czy to jest na pewno button a nie cokolwiek z tekstem :)